.travis.yml

```diff
@@ -1,8 +1,20 @@
-language: node_js
 sudo: false
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
I [think you're right](https://github.com/greenkeeperio/greenkeeper/issues/222).